MediaRecorder.html

```diff
@@ -148,12 +148,21 @@
      </ol>
     </dd>
 
-    <dt>static DOMString canRecordMimeType(DOMString mimeType)</dt>
-    <dd><p>This method <em title="must" class="rfc2119">must</em> return the empty string if <code>mimeType</code> is a type that the user agent knows it cannot record; it <em title="must" class="rfc2119">must</em> return "probably" if the user agent is confident that <code>mimeType</code> represents a type that it can record; and it <em title="must" class="rfc2119">must</em> return "maybe" otherwise. Implementors are encouraged to return "maybe" unless the type can be confidently established as being supported or not.</p>
+    <dt>static bool isTypeSupported(DOMString type)</dt>
+    <dd> Check to see whether a <code>MediaRecorder</code> can record in the specified Mime type. If true is returned from this method, it only indicates that the <code>MediaRecorder</code> implementation is capable of recording Blob objects for the specified MIME type. Recording may still fail if sufficient resources are not available to support the concrete media encoding. When this method is invoked, the user agent must run the following steps:
+     <ol class="method-algorithm">
+      <li>If <code>type</code> is an empty string, then return false.</li>
+      <li>If <code>type</code> does not contain a valid MIME type string, then return false.</li>
+      <li>If <code>type</code> contains a media type or media subtype that the MediaSource does not support, then return false.</li>
+      <li>If <code>type</code> contains a media container that the MediaSource does not support, then return false.</li>
+      <li>If <code>type</code> contains a codec that the MediaSource does not support, then return false.</li>
```

Comment:
Oh, I actually missed that. It seems like that line enough would be enough as the definition, since it encompasses all other problems. Editorial only, though.